src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java

```diff
@@ -81,21 +81,22 @@ public void adsVuapVideoClosesWhenTapCloseButtonOasis(Page page, String slotName
           groups = "AdsVuapCheckSlotSizesOasis"
   )
   public void adsVuapCheckSlotSizesOasis(Page page, String slotName, String iframeId) throws InterruptedException {
+    String slotSelector = AdsContent.getSlotSelector(slotName);
     AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page), DESKTOP_SIZE);
     scrollToSlot(slotName, ads);
     VideoFanTakeover videoFanTakeover = new VideoFanTakeover(driver, iframeId);
```

Comment:
In this case I have to define this variable after the scroll to find expected iframe